tester/runner.go

```diff
@@ -329,23 +330,30 @@ func (r *Runner) runTests(ctx context.Context, txn storage.Transaction, enablePr
 			return nil, err
 		}
 
-		// Aggregate the bundle modules with other ones provided
-		if r.modules == nil {
-			r.modules = map[string]*ast.Module{}
-		}
 		for path, b := range r.bundles {
 			for name, mod := range b.ParsedModules(path) {
-				r.modules[name] = mod
+				bundleModules[name] = mod
 			}
 		}
 	}
 
-	if r.modules != nil && len(r.modules) > 0 {
+	if r.modules == nil {
+		r.modules = map[string]*ast.Module{}
```

Comment:
`len(r.modules) == 0` for `r.modules == nil`, right? So we could move this to an `else` of the following `if`?

Reply:
Sounds good. I'll read the code again, I think of we're only operating with bundles or files (exclusively), we don't even need r.modules in the bundle case. But I'll need another look.